Foundation/NSCalendar.swift

```diff
@@ -163,7 +163,10 @@ public class NSCalendar : NSObject, NSCopying, NSSecureCoding {
     }
     
     public func copyWithZone(_ zone: NSZone) -> AnyObject {
-        NSUnimplemented()
+        let copy = NSCalendar(calendarIdentifier: calendarIdentifier)!
+        copy.locale = locale
+        copy.timeZone = timeZone
+        return copy
```

Comment:
I think the question was if a consumer of the api mutates those values via assigning to the setter of firstWeekday etc, the copy should reflect those mutations.

It seems to be missing
```swift
copy.firstWeekday = firstWeekday
copy.minimumDaysInFirstWeek = minimumDaysInFirstWeek
copy._startDate = _startDate
```